demo/nuxeo-html-editor/index.html

```diff
@@ -28,6 +28,7 @@
     <!-- nuxeo elements -->
     <link rel="import" href="../nuxeo-demo.html">
     <script src="../../widgets/alloy/alloy-editor-all.js"></script> <!-- NXP-23147: explicitly import alloy -->
+    <!-- <script src="../../widgets/quill/quill.min.js"></script> --> <!-- ELEMENTS-1323: import Quill to use it instead-->
```

Comment:
The idea would be for people do add this import to their custom bundle, right?

Reply:
was planning on having a configuration property and loading one or the other in the `index.jsp` but open for discussion at this point

Reply:
If we would expose such a property, I hope it wouldn't be expected to carry over for LTS2021? 😬